### PR TITLE
Spanish translation updated to version 2.9.24

### PR DESCRIPTION
**Pull request by FredMan95**

### CODE REVIEW

**Comment by LagradOst** [comment]
ERROR:/home/runner/work/CloudStream-3/CloudStream-3/app/src/main/res/values-es/strings-es.xml: Resource and asset merger: ' ' is not a valid resource name character

**Comment by FredMan95** [comment]
I have translated all the strings and I don't know why this error occurs.

Do you have a location management platform such as Crowdin?

File: app/src/main/res/values-es/strings-es.xml [hunk 1]
```diff
@@ -1,189 +1,521 @@
...
+    <string name="status_ongoing">En curso</string> 
+    <string name="status_completed">Terminado</string> 
+    <string name="status">Estado</string> 
+    <string name="year">Año</string> 
+    <string name="rating">Clasificación</string> 
+    <string name="duration">Duración</string> 
+    <string name="site">Sitio</string> 
+    <string name="synopsis">Sinopsis</string> 
+ 
+    <string name="Puesto en cola">Puesto en cola</string>
```
Comment (LagradOst) [inline app/src/main/res/values-es/strings-es.xml:302]: It is this string that causes compile error

Comment (FredMan95) [reply app/src/main/res/values-es/strings-es.xml:302]: Should I resend the pull request?

Comment (Blatzar) [reply app/src/main/res/values-es/strings-es.xml:302]: Just change the line to make it work and it'll update here too :+1:

Comment (Blatzar) [reply app/src/main/res/values-es/strings-es.xml:302]: Btw i think the error here is that name doesn't take anything with spaces, delete the line or replace the name with what's used in other languages

<string name="**Puesto en cola**">....

Comment (FredMan95) [reply app/src/main/res/values-es/strings-es.xml]: Done

**Comment by LagradOst** [comment]
LGTM, ready to merge? 👍

**Pull request merged**